src/QMCWaveFunctions/RotatedSPOs.cpp

```diff
@@ -171,11 +205,21 @@ void RotatedSPOs::writeVariationalParameters(hdf_archive& hout)
     if (rows > 0)
       cols = history_params_[0].size();
 
-    Matrix<ValueType> tmp(rows, cols);
+#ifdef QMC_COMPLEX
+    size_t flattened_cols = 2 * cols;
+    Matrix<RealType> tmp(rows, flattened_cols);
+    for (size_t i = 0; i < rows; i++)
+      for (size_t j = 0; j < cols; j++)
+      {
+        tmp(i, j)        = std::real(history_params_[i][j]);
+        tmp(i, j + cols) = std::imag(history_params_[i][j]);
+      }
+#else
+    Matrix<RealType> tmp(rows, cols);
     for (size_t i = 0; i < rows; i++)
       for (size_t j = 0; j < cols; j++)
         tmp(i, j) = history_params_[i][j];
-
+#endif
     std::string rot_hist_name = std::string("rotation_history_") + SPOSet::getName();
     hout.write(tmp, rot_hist_name);
```

Comment:
I think IO should remain in ValueType.

Reply:
Doing it this way made it so that the parameters written by both global vs. history method were consistent with the what is written to "name_value_lists" in the vp.h5 which is written by VariableSet, which is now strictly reals. Was easier to deal with as we were debugging and testing derivatives, but can change if necessary

Reply:
`Matrix<Complex>` will be stored as (nrows, ncols, 2). it is a lot safer than (ncrows, ncols*2). Once the ordering of real/complex in myVars are adjusted, this place will converge to what I was asking for.